projects/demo/src/app/app.component.ts

```diff
@@ -127,6 +127,10 @@ export class AppComponent implements OnInit {
     }
   }
   public myTitle: string;
+  onBlurMethod()
+  {
+    console.log(this.myTitle);
+  }
```

Comment:
Is this required?

Reply:
Yes it is required, otherwise it will cause this error "Property 'onBlurMethod' does not exist on type 'AppComponent'."